### ISSUE TITLE
Browse call number preview and select buttons are out of drawn boxes

### ISSUE THREAD

**Issue by saseestone** [sul-dlss/SearchWorks#3742]
Staff have reported that when browsing the "001AAC4737" call number on this record: https://searchworks.stanford.edu/view/23684, the "Preview" and "Select" action buttons appear outside the drawn boxes.

![image](https://github.com/sul-dlss/SearchWorks/assets/14897546/cc634d6b-03f0-4781-8520-67aecd6ffe6f)

It appears that this happens if a user selects a second or third option in the "Start at call number:" section: https://github.com/sul-dlss/SearchWorks/assets/14897546/c0b83b26-a1a5-4893-90ee-48ef96e701d7

Could this be related to the Bootstrap update?

**Comment by jcoyne** [comment]
Do we have designs for how we want this to look?

**Comment by cbeer** [comment]
We used to truncate titles:

![Image](https://github.com/sul-dlss/SearchWorks/assets/111218/eaf3101d-b029-4f1a-b8cc-28a60becf129)

Presumably this was a regression from the bootstrap upgrade.

**Comment by saseestone** [comment]
Noting that if the record has only one call number to browse, our design system seems correct. It's if the record has more than one call number, and the user selects one of those subsequent call numbers to browse.

**Comment by dbranchini** [comment]
Note the select should not be a button, but instead a checkbox too.

**Comment by jcoyne** [comment]
This was caused by a javascript error:
```
Uncaught TypeError: val.first is not a function
    <anonymous> debugger eval code:1
    showPreview https://searchworks-stage.stanford.edu/assets/application-ce5f50777a4d78ab7ee9f8472cd77c5fa83056f8809523aade145a6d4e3aa0a5.js:16455
    jQuery 2
    showPreview https://searchworks-stage.stanford.edu/assets/application-ce5f50777a4d78ab7ee9f8472cd77c5fa83056f8809523aade145a6d4e3aa0a5.js:16454
    jQuery 3
    plugContentAndPlugins https://searchworks-stage.stanford.edu/assets/application-ce5f50777a4d78ab7ee9f8472cd77c5fa83056f8809523aade145a6d4e3aa0a5.js:16324
    fetchContentViaAjaxAndInsert https://searchworks-stage.stanford.edu/assets/application-ce5f50777a4d78ab7ee9f8472cd77c5fa83056f8809523aade145a6d4e3aa0a5.js:16307
    jQuery 6
    fetchContentViaAjaxAndInsert https://searchworks-stage.stanford.edu/assets/application-ce5f50777a4d78ab7ee9f8472cd77c5fa83056f8809523aade145a6d4e3aa0a5.js:16301
    checkCache https://searchworks-stage.stanford.edu/assets/application-ce5f50777a4d78ab7ee9f8472cd77c5fa83056f8809523aade145a6d4e3aa0a5.js:16297
    append https://searchworks-stage.stanford.edu/assets/application-ce5f50777a4d78ab7ee9f8472cd77c5fa83056f8809523aade145a6d4e3aa0a5.js:16344
    showPreview https://searchworks-stage.stanford.edu/assets/application-ce5f50777a4d78ab7ee9f8472cd77c5fa83056f8809523aade145a6d4e3aa0a5.js:16452
    openThisBrowser https://searchworks-stage.stanford.edu/assets/application-ce5f50777a4d78ab7ee9f8472cd77c5fa83056f8809523aade145a6d4e3aa0a5.js:16440
    jQuery 40
        complete
        c
        fireWith
        u
        tick
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        dt
        start
        timer
        yt
        a
        dequeue
        queue
        each
        each
        queue
debugger eval code:1:5
```

**Comment by jcoyne** [comment]
There were several problems here. One was some bad javascript fixed in #3752.

Another is that the titles are no longer being truncated the same way here:
https://github.com/sul-dlss/SearchWorks/blob/main/app/views/catalog/_index_header_gallery.html.erb#L20
